app/services/generaldetails/CandidateDetailsService.scala

```diff
@@ -48,22 +49,27 @@ trait CandidateDetailsService {
     val updatePersonalDetailsFut = pdRepository.update(applicationId, userId, personalDetails,
       sourceStatuses = List(CREATED, IN_PROGRESS), targetStatus = IN_PROGRESS)
     val contactDetailsFut = cdRepository.update(userId, contactDetails)
+    val fastPassDetailsFut = fpdRepository.update(applicationId, candidateDetails.fastPassDetails)
```

Comment:
As discussed my initial thought was to include fast pass updates in personal repo. But we might need to use the fast pass details else where may be reporting. So Leaving as it.